### ISSUE TITLE
The model of 99.53% on lfw

### ISSUE THREAD

**Issue by wuqiangch** [wuyuebupt/LFWBenchmark#1]
@wuyuebupt Can you share the model of  99.53% on lfw?  And what's the database you used?

**Comment by wuyuebupt** [comment]
@wuqiangch Will add the model soon. The database is MS-Celeb-1M (MS-Celeb-1M: A Dataset and Benchmark for Large-Scale Face Recognition, ECCV 16).

**Comment by wuqiangch** [comment]
@wuyuebupt Thanks!

**Comment by wuyuebupt** [comment]
@wuqiangch The model is at https://drive.google.com/file/d/0B0W0rPKOeIBydzgzRTZ1aktrN1k/view?usp=sharing .

**Comment by wuqiangch** [comment]
@wuyuebupt Can you send it to me by email? or baidu pan?You kown wo can't get on goole in China,haha. Thanks！

**Comment by wuqiangch** [comment]
@wuyuebupt  I can download it now Thanks！

**Comment by wuqiangch** [comment]
@wuyuebupt There are more than 1,000 identities' overlap between MS-Celeb-1M and LFW.Have you remove the identities' of overlap to train the model and test on lfw？

**Comment by wuqiangch** [comment]
@wuyuebupt What't the method you used to detect face landmarks?

**Comment by wuyuebupt** [comment]
@wuqiangch Nope, I directly used the data. About alignment, for testing, I use MTCNNv2(https://github.com/kpzhang93/MTCNN_face_detection_alignment).  For training, no alignment was used since the faces are aligned by Microsoft. Using the same alignment method with testing will get higher accuracy but 99.5% is enough.

**Comment by wuqiangch** [comment]
OK，thanks.


发自网易邮箱大师
On 03/01/2017 22:18, WU Yue wrote:

@wuqiangch Nope, I directly used the data. About alignment, for testing, I use MTCNNv2(https://github.com/kpzhang93/MTCNN_face_detection_alignment). For training, no alignment was used since the faces are aligned by Microsoft. Using the same alignment method with testing will get higher accuracy but 99.5% is enough.
